addons/website_event_track/views/event_event_views.xml

```diff
@@ -15,11 +15,9 @@
                     <field name="track_count" string="Tracks" widget="statinfo"/>
                 </button>
             </xpath>
-            <xpath expr="//field[@name='website_menu']" position="after">
-                <label for="website_track" string="Showcase Tracks"/>
-                <field name="website_track"/>
-                <label for="website_track_proposal" string="Allow Track Proposals"/>
-                <field name="website_track_proposal"/>
+            <xpath expr="//group[@name='sub_menu_left']" position="inside">
+                <field name="website_track" string="Talks"/>
+                <field name="website_track_proposal" string="Talk Proposal"/>
```

Comment:
Why did we change the string any reason?

Reply:
Hello @thsh-odoo 
I changed them according to the mock-up.

Thanks :smile: